src/test/java/com/rakovets/course/java/core/practice/oop_principles/battle_ground/TestArcher.java

```diff
@@ -0,0 +1,27 @@
+package com.rakovets.course.java.core.practice.oop_principles.battle_ground;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestArcher {
+    Archer archer = new Archer("Archer", 100, true);
+    Enemy enemy = new EnemySkeleton(100, true);
+
+    @Test
+    public void testAttackEnemy() {
+        final int EXPECTED_ENEMY_HEALTH = 90;
+
+        archer.attackEnemy(enemy);
+
+        Assertions.assertEquals(EXPECTED_ENEMY_HEALTH, enemy.getHealth());
```

Comment:
```suggestion
        archer.attackEnemy(enemy);

        Assertions.assertEquals(90, enemy.getHealth());
```

Reply:
Magic numbers?))

At the current stage, it's rather difficult for me to define where can I use magic numbers and where it's bad practice.
I think it will come with experience)